includes/discovery/functions.inc.php

```diff
@@ -1394,3 +1394,58 @@ function find_port_id($description, $identifier = '', $device_id = 0, $mac_addre
 
     return (int) dbFetchCell($sql, $params);
 }
+
+/**
+ * function specific for Jetstream OS, find port_id from DB
+ *
+ * @param  string  $description  matched against ifDescr in various forms
+ * @param  int  $device_id  restrict search to ports on a specific device
+ * @return int
+ */
+function find_jetstream_port_id($description = '', $device_id = 0)
+{
+    if (! ($device_id || $description)) {
+        return 0;
+    }
+
+    //jetstream specific port name variations
+    $pn = [];
+    $pn[] = 'gigabitEthernet ' . $description;
+    $pn[] = 'gigabitEthernet ' . $description . ' : copper';
+    $pn[] = 'gigabitEthernet ' . $description . ' : fiber';
+
+    $res = \App\Models\Port::where('device_id', $device_id)->whereIn('ifDescr', $pn)->first('port_id');
```

Comment:
first('port_id') should be value('port_id')

Reply:
done